### ISSUE TITLE
Add support for Gradle Managed Virtual Devices

### ISSUE THREAD

**Issue by DanielJette** [ndtp/android-testify#7]
**Is your feature request related to a problem? Please describe.**

Add support for Gradle Managed Virtual Devices

https://developer.android.com/studio/preview/features/#gmd

This feedback relates to:
- [x] The Kotlin library
- [x] The Gradle plugin
- [x] The IntelliJ Platform plugin
- [ ] The sample code
- [ ] The documentation

**Comment by DanielJette** [comment]
https://github.com/ndtp/android-testify/tree/7-gradle-managed-virtual-devices

**Comment by kpateldev** [comment]
Looking forward to support for gradle-managed devices and the ability to record and run screenshot tests using this feature. Thanks!

**Comment by olegosipenko** [comment]
In my company we built our screenshot testing pipeline on GMD, so I can help with adding the support for it.
The main issue here is how to get images from GMD when we want to generate baselines or to pull diffs. Typically we use ADB for this, but here one can't connect to GMD via ADB, because Gradle shuts it down after test execution. So, the solution I found in Android samples is to use [TestStorageService](https://developer.android.com/reference/androidx/test/services/storage/TestStorage) 
To enable it we need to add
```
android {
   defaultConfig {
       testInstrumentationRunnerArguments += mapOf("useTestStorageService" to "true")
   }
}

androidTestUtil("androidx.test.services:test-services:$servicesVersion") 
```
to `build.gradle` and then in the screenshot capturing method use this service to store image using extenstion method from the Espresso
```
fun Bitmap.writeToTestStorage(name: String) {
  writeToTestStorage(TestStorage(), name)
}
```
After test execution screenshots would be available in `build/output` folder

So, my proposal is to create a new TestifyFeature and change the capture method analogous to other alternative capture methods.
